proto/aquila/transfer.proto

```diff
@@ -16,6 +16,17 @@ message InformationResponse {
   bool success = 1;
 }
 
+message ActionExecuteRequest {
```

Comment:
Aquila sends paramters to Action (if Aquila has paramters) 
Stream init. by the Action

Reply:
Besides the `execution_identifier` we need to provide an identification for the runnable action function otherwise the action doesn't know which function to execute. The `execution_identifier` is just an internal identifier, which we use to link a certain execution call.